src/actions/PetitionsActions.js

```diff
@@ -19,14 +19,15 @@ export function fetchPetitions ({ location, params }) {
     page: parseInt(params && params.page || query.page || 1),
     city: params && params.city || query.city || '',
     cityName: params && params.cityName || query.cityName || '',
-    limit: parseInt(query.limit || 12)
+    limit: parseInt(query.limit || 12),
+    sort: params && params.sort || query.sort || ''
```

Comment:
oh, I guess I don’t need the tests for params. Sorting will only be done with query parameters